src/modules/launcher/PowerLauncher/MainWindow.xaml.cs

```diff
@@ -144,6 +146,15 @@ private double WindowLeft()
             return left;
         }
 
+        private double WindowTop()

```

Comment:
sorry for commenting out of context, does WindowTop() get the window position, and calculates the Y coordinate using dip1 and dip2? Some comments on the function would be helpful to facilitate a walkthrough of the function :) I will run your branch on my side to test that it launches on the top center of the screen :)

Reply:
@laviusmotileng-ms Sure! Thanks for suggestion.